test/unary_test.js

```diff
@@ -5,11 +5,15 @@ import * as unaryFunctions from '../src/unary.js';
 import * as utils from './utils.js';
 
 describe('test unary', function() {
-  function testUnary(op, input, expected, shape) {
+  function testUnary(op, input, expected, shape, atolTolerance) {
```

Comment:
*atol* stands for *absolute tolerance*, and So *atolTolerance* would means *absolute tolerance tolerance*. How about either `absoluteTolerance` or just `atol`?